.mvn/develocity.xml

```diff
@@ -1,24 +1,21 @@
-<gradleEnterprise>
+<develocity>
+  <projectId>james</projectId>
   <server>
-    <url>https://ge.apache.org</url>
+    <url>https://develocity.apache.org</url>
     <allowUntrusted>false</allowUntrusted>
     <!--
-    credentials provided by jenkins exposing GRADLE_ENTERPRISE_ACCESS_KEY
+    credentials provided by jenkins exposing DEVELOCITY_ACCESS_KEY
     apache members can create their own on the server
     -->
   </server>
   <!-- options are documented at https://docs.gradle.com/enterprise/maven-extension/#configuring_background_uploading -->
   <!-- chosen values align with https://cwiki.apache.org/confluence/display/INFRA/Project+Onboarding+Instructions+for+Develocity -->
   <buildScan>
     <backgroundBuildScanUpload>#{isFalse(env['CI'])}</backgroundBuildScanUpload>
-    <publishIfAuthenticated>true</publishIfAuthenticated>
     <!-- Always publish build scans for CI, only if requested locally differs from onboarding instructions -->
-    <publish>#{env['CI'] == null ? 'ON_DEMAND' : 'ALWAYS'}</publish>
-    <capture>
-      <goalInputFiles>true</goalInputFiles>
```

Comment:
I know these are the default values but  do you mind putting the configuration back ? 
it allows to easily change it and it is more explicit about what is captured which could be important when publishing local results

Reply:
No problem, done in the latest push